test/e2e/kubeproxy.go

```diff
@@ -38,7 +40,6 @@ import (
 const (
 	endpointHttpPort        = 8080
 	endpointUdpPort         = 8081
-	endpointHostPort        = 8082
```

Comment:
why was this removed ?

also, if you have good reason to remove it, we should do it in a separate PR..